### PR TITLE
Hsdo 391 v1

### PR DESCRIPTION
**Pull request by josephgknox**
Whatcha think?! Did we make a thing?

https://sites.stanford.edu/sws-toggle-block/toggle-block-example

### CODE REVIEW

File: js/stanford-toggle-block.js [hunk 1]
```diff
@@ -1,70 +1,40 @@
 Drupal.behaviors.stanford_toggle_block = {
   attach: function(context, settings) {
 
-  /**
```
Comment (sherakama) [inline js/stanford-toggle-block.js]: I'm beginning to think you don't like my code comments.

Y uuz always deleting them?

Comment (jbickar) [inline js/stanford-toggle-block.js]: Y U HATE FUTURE US???

**Comment by sherakama** [comment]
@josephgknox ,

Two comments on the styles.

1. Too much padding/margin at the bottom. Big chunk of brown below the white border.
2. The transition bleeds outside of the white frame. Can you contain it?

One comment on the PR.

1. Can you restore the code comments in the js file please.

File: js/stanford-toggle-block.js (hunk 1 above)
Comment (josephgknox) [inline js/stanford-toggle-block.js]: I didn't touch your comments (intentionally, that is) :)

Could this have happened upon my exporting of the Feature!? How can I avoid this happening? It looks like my commit was from when I had done just that. I was working on the latest... In any case. I'll add the top comment back. Are there any others (outside of those that I collated) that need to be rolled back in?

Comment (sherakama) [inline js/stanford-toggle-block.js]: The export should not remove comments. I think you may have been missing the latest when you exported. 

Any which way, take a look at this diff and restore them all. There are 12 separate comment blocks that got wiped.

Comment (josephgknox) [inline js/stanford-toggle-block.js]: Ok, they're back. Let's keep them there this time, Joe.

**Comment by sherakama** [comment]
Thank you!

**Pull request merged**